src/EditorFeatures/VisualBasicTest/EditAndContinue/TopLevelEditingTests.vb

```diff
@@ -3943,7 +3943,8 @@ End Class
                         {
                             SemanticEdit(SemanticEditKind.Update, Function(c) c.GetMember("C.F1"))
                         })
-                })
+                },
+                capabilities:=EditAndContinueCapabilities.AddMethodToExistingType)
```

Comment:
Is it known why these tests originally passed in #62364?

Reply:
No idea.

Reply:
Thank you for fixing this. Seems like my last push didn't kick off a build for some reason.